### ISSUE TITLE
[Truffle] Missing method needed for CGI

### ISSUE THREAD

**Issue by bjfish** [jruby/jruby#2801]
`define_method` is used to implement CGI:
Example
```ruby
module Example
     def nOE_element_def(attributes = {}, &block)
      puts "Hello"
    end

    instance_method(:nOE_element_def).tap do |m|
      # - O EMPTY
      for element in %w[ IMG BASE BASEFONT BR AREA LINK PARAM HR INPUT
          ISINDEX META ]
        define_method(element.downcase, m)
      end
    end
end
```
Error
```
rang.rb:2087:in `define_method': Truffle doesn't have a case for the org.jruby.truffle.nodes.core.ModuleNodesFactory$DefineMethodNodeFactory$DefineMethodPolymorphicNode node with values of type  Module(org.jruby.truffle.runtime.core.RubyModule) java.lang.String UnboundMethod(org.jruby.truffle.runtime.core.RubyUnboundMethod) org.jruby.truffle.runtime.UndefinedPlaceholder (TypeError)
``

**Comment by bjfish** [comment]
Tap added at 0e4e50c5d9cedd1da6ffbadb13a88b0623a1be54

**Comment by chrisseaton** [comment]
`define_method` fixed for `UnboundMethod` in c6599ead5585be5a89e74dfa56b29eb8d3b3e8d3.
